### PR TITLE
proofread: alluxio

### PR DESCRIPTION
**Pull request by rayw000**
close: https://github.com/juicedata/docs/issues/100

Hi @AIXjing , could you please review this PR when you have time? Thank you!

### CODE REVIEW

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ slug: /comparison/juicefs_vs_alluxio
 
 # JuiceFS vs. Alluxio
 
-[Alluxio](https://www.alluxio.io) (/əˈlʌksio/) is a data access layer in the big data and machine learning ecosystem. Initially as research project "Tachyon", it was created at the University of California, Berkeley's [AMPLab](https://en.wikipedia.org/wiki/AMPLab) as creator's Ph.D. thesis in 2013. Alluxio was open sourced in 2014.
+[Alluxio](https://www.alluxio.io) (/əˈlʌksio/) is a data access layer in the big data and machine learning ecosystem. It was a research project "Tachyon" initially created in 2013 as the Ph.D. thesis of the creator of the [AMPLab](https://en.wikipedia.org/wiki/AMPLab) in the University of California Berkeley. Alluxio was open sourced in 2014.
```
Comment (AIXjing) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:7]: 这里貌似可以不用改，原来的好像摘自wiki: https://en.wikipedia.org/wiki/Alluxio

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 2]
```diff
@@ -29,48 +29,48 @@ The following table shows difference of main features between Alluxio and JuiceF
...
+A single file is [stored](../../reference/how_juicefs_store_files.md) in JuiceFS in three levels: chunk, slice and block. A file will be split into multiple blocks, and be compressed and encrypted (optional) before it is stored into object storage.
 
-Alluxio stores file as object to UFS. The file doesn't be split info blocks like JuiceFS does.
+Alluxio stores files as _objects_ into UFS. It doesn't split files info blocks like what JuiceFS does.
 
 ### Cache granularity
 
-The [default block size](../../reference/how_juicefs_store_files.md) of JuiceFS is 4MiB, compare to 64MiB of Alluxio, the granularity is smaller. The smaller block size is better for random read (e.g. Parquet and ORC) workload, i.e. cache management will be more efficiency.
+The [default block size](../../reference/how_juicefs_store_files.md) of JuiceFS is 4MiB, compare to 64MiB of Alluxio, the granularity is smaller. Smaller block size is better for random read (e.g. Parquet and ORC) workload, i.e. cache management will be more efficiency.
```
Comment (AIXjing) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:38]:
```suggestion
The [default block size](../../reference/how_juicefs_store_files.md) of JuiceFS is 4MiB, and thus its granularity is smaller compared to 64MiB of Alluxio. Smaller block size is better for random read (e.g. Parquet and ORC) workload, i.e. cache management will be more efficiency.
```

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 3]
```diff
@@ -29,48 +29,48 @@ The following table shows difference of main features between Alluxio and JuiceF
...
 
 ### Hadoop-compatible
 
-JuiceFS is [HDFS-compatible](../../deployment/hadoop_java_sdk.md). Not only compatible with Hadoop 2.x and Hadoop 3.x, but also variety of components in Hadoop ecosystem.
+JuiceFS is [HDFS-compatible](../../deployment/hadoop_java_sdk.md). Not only compatible with Hadoop 2.x and Hadoop 3.x, but also compatible for various components in Hadoop ecosystem.
 
 ### Kubernetes CSI Driver
 
-JuiceFS provides [Kubernetes CSI Driver](https://github.com/juicedata/juicefs-csi-driver) to help people who want to use JuiceFS in Kubernetes. Alluxio provides [Kubernetes CSI Driver](https://github.com/Alluxio/alluxio-csi) too, but this project seems like not active maintained and not official supported by Alluxio.
+JuiceFS provides [Kubernetes CSI Driver](https://github.com/juicedata/juicefs-csi-driver) to help people who want to use JuiceFS in Kubernetes. Alluxio provides [Kubernetes CSI Driver](https://github.com/Alluxio/alluxio-csi) too, but this project seems like not being active maintained and not being official supported by Alluxio.
```
Comment (AIXjing) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:46]:
```suggestion
JuiceFS provides [Kubernetes CSI Driver](https://github.com/juicedata/juicefs-csi-driver) to help people who want to use JuiceFS in Kubernetes. Alluxio provides [Kubernetes CSI Driver](https://github.com/Alluxio/alluxio-csi) too, but this project seems not active and is not officially supported by Alluxio.
```

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 4]
```diff
@@ -29,48 +29,48 @@ The following table shows difference of main features between Alluxio and JuiceF
...
 
 ### Atomic metadata operation
 
-A metadata operation in Alluxio has two steps: the first step is modify state of Alluxio master, the second step is send request to UFS. As you can see, the metadata operation isn't atomic, its state is unpredictable when the operation is executing or any failure occurs. Alluxio relies on UFS to implement metadata operations, for example rename file operation will become copy and delete operations.
+A metadata operation in Alluxio consists two steps: the first step is to modify the state of Alluxio master, the second step is to send request to UFS. From which we can see that the metadata operation isn't atomic. The state is unpredictable when the operation is executing or falls into any failures. Alluxio requires UFS to implement metadata operations, for example rename file operation will become copy and delete operations.
```
Comment (AIXjing) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:54]:
```suggestion
A metadata operation in Alluxio consists of two steps: the first step is to modify the state of Alluxio master, and the second step is to send request to UFS. Thus, we can see that the metadata operation isn't atomic. The state is unpredictable when the operation is being executed or any failures happen. Alluxio requires UFS to implement metadata operations, for example, rename file operation will become copy and delete operations.
```

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 5]
```diff
@@ -29,48 +29,48 @@ The following table shows difference of main features between Alluxio and JuiceF
...
+Thanks to [Redis transaction](https://redis.io/topics/transactions), **most of the metadata operations of JuiceFS are atomic**, e.g. rename file, delete file, rename directory. You don't have to worry about the consistency and performance.
 
 ### Consistency
 
-Alluxio loads metadata from the UFS as needed and it doesn't have information about UFS at startup. By default, Alluxio expects that all modifications to UFS occur through Alluxio. If changes are made to UFS directly, you need to sync metadata between Alluxio and UFS either manually or periodically. As ["Atomic metadata operation"](#atomic-metadata-operation) section says, the two steps of metadata operation may result in inconsistency.
+Alluxio loads metadata from the UFS as needed. It doesn't have information about UFS at startup. By default, Alluxio expects that all modifications on UFS are done through Alluxio. If changes on UFS are made directly, you'll need to sync metadata between Alluxio and UFS either manually or periodically. As ["Atomic metadata operation"](#atomic-metadata-operation) section says, the two steps of metadata operation may result in inconsistency.
```
Comment (AIXjing) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:60]:
```suggestion
Alluxio loads metadata from the UFS as needed. It doesn't have information about UFS at startup. By default, Alluxio expects that all modifications on UFS are completed through Alluxio. If changes are made directly on UFS, you'll need to sync metadata between Alluxio and UFS either manually or periodically. As we have mentioned in ["Atomic metadata operation"](#atomic-metadata-operation) section, the two-step metadata operation may result in inconsistency.
```

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 6]
```diff
@@ -29,48 +29,48 @@ The following table shows difference of main features between Alluxio and JuiceF
...
 
-JuiceFS provides strong consistency, in both metadata and data. **The metadata service of JuiceFS is the single source of truth, not a mirror of UFS.** The metadata service doesn't rely on object storage to obtain metadata. Object storage just be treated as an unlimited block storage. There isn't any inconsistency between JuiceFS and object storage.
+JuiceFS provides strong consistency for both metadata and data. **The metadata service of JuiceFS is the single source of truth, not a mirror of UFS.** The metadata service doesn't rely on object storage to obtain metadata. Object storage just be treated as an unlimited block storage. There isn't any inconsistency between JuiceFS and object storage.
```
Comment (AIXjing) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:62]:
```suggestion
JuiceFS provides strong consistency for both metadata and data. **The metadata service of JuiceFS is the single source of truth, not a mirror of UFS.** The metadata service doesn't rely on object storage to obtain metadata, and object storage is just treated as unlimited block storage. Thus, there will not be any inconsistency between JuiceFS and object storage.
```

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 7]
```diff
@@ -29,48 +29,48 @@ The following table shows difference of main features between Alluxio and JuiceF
...
 
 ### Data encryption
 
-JuiceFS supports data encryption in transit and at rest. Alluxio community edition doesn't have this feature, but [enterprise edition](https://docs.alluxio.io/ee/user/stable/en/operation/Security.html#end-to-end-data-encryption) has.
+JuiceFS supports data encryption in transit and at rest. Alluxio community edition doesn't provide this feature, while [enterprise edition](https://docs.alluxio.io/ee/user/stable/en/operation/Security.html#end-to-end-data-encryption) does.
 
 ### Zero-effort operation
 
-Alluxio's architecture can be divided into 3 components: master, worker and client. A typical cluster consists of a single leading master, standby masters, a job master, standby job masters, workers, and job workers. You need operation these masters and workers by yourself.
+Alluxio's architecture can be divided into 3 components: master, worker and client. A typical cluster consists of a single leading master, standby masters, a job master, standby job masters, workers, and job workers. You need to maintain these masters and workers by yourself.
```
Comment (AIXjing) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:74]:
```suggestion
Alluxio's architecture can be divided into 3 components: master, worker and client. A typical cluster consists of a single leading master, standby masters, a job master, standby job masters, workers, and job workers. You need to maintain all these masters and workers by yourself.
```

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 8]
```diff
@@ -29,48 +29,48 @@ The following table shows difference of main features between Alluxio and JuiceF
...
 
-JuiceFS uses Redis or [others](../../guide/how_to_setup_metadata_engine.md) as the metadata engine. You could use service managed by public cloud provider easily as JuiceFS's metadata engine. There isn't any operation needed.
+JuiceFS uses Redis or [others](../../guide/how_to_setup_metadata_engine.md) as the metadata engine. You could easily use service managed by public cloud provider as the JuiceFS's metadata engine. There isn't any operation needed.
```
Comment (AIXjing) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:76]:
```suggestion
JuiceFS uses Redis or [others](../../guide/how_to_setup_metadata_engine.md) as the metadata engine. You could easily use the service managed by public cloud provider as JuiceFS's metadata engine without the need for operation.
```

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md (hunk 1 above)
Comment (Suave) [reply docs/en/introduction/comparison/juicefs_vs_alluxio.md:7]: yes, we don't change quote messages

Comment (rayw000) [reply docs/en/introduction/comparison/juicefs_vs_alluxio.md:7]:
```suggestion
[Alluxio](https://www.alluxio.io) (/əˈlʌksio/) is a data access layer in the big data and machine learning ecosystem. Initially as research project "Tachyon", it was created at the University of California, Berkeley's [AMPLab](https://en.wikipedia.org/wiki/AMPLab) as creator's Ph.D. thesis in 2013. Alluxio was open sourced in 2014.
```
restore quoted messages

**Pull request merged**